src/Helpers/SpellHelper.php

```diff
@@ -56,6 +57,12 @@ public function getSpells(): Collection
 
     public function getSelfSpells(): Collection
     {
+        $raceName = app(SelectorService::class)->getUserSelectedDominion()->race->name;
```

Comment:
I don't really like that we're using the SelectorService for this. We should continue the discussion later about how to tackle this properly, imo.

Reply:
Yeah I know. We can discuss this tonight perhaps?

Reply:
Please pass a `Race` instance into this method.

Reply:
Done!